visual-java/src/main/java/com/saucelabs/visual/VisualApi.java

```diff
@@ -25,6 +25,13 @@
 public class VisualApi {
   private static final Logger log = LoggerFactory.getLogger(VisualApi.class);
 
+  public static String resolveEndpoint(String endpoint) {
```

Comment:
do we need to expose this option here? It doesn't look like a correct domain for VisualApi

Reply:
right, this can be certainly made private